google/cloud/aiplatform/metadata/artifact.py

```diff
@@ -262,12 +263,13 @@ def create(
         project: Optional[str] = None,
         location: Optional[str] = None,
         credentials: Optional[auth_credentials.Credentials] = None,
+        base_artifact: Optional[types_base.BaseArtifactSchema] = None,
     ) -> "Artifact":
         """Creates a new Metadata Artifact.
 
         Args:
             schema_title (str):
-                Required. schema_title identifies the schema title used by the Artifact.
+                Optional. schema_title identifies the schema title used by the Artifact.
```

Comment:
We probably should add a comment to mentioned that one of `schema_title` and `base_artifact` needs to be set. And also what would the behavior be if both field are set and has conflict info.

Reply:
Done.